spec/models/education_spec.rb

```diff
@@ -1,5 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe Education, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'has a valid factory' do
+    expect(FactoryGirl.build(:education)).to be_valid
+  end
+
+  describe 'converts date to strftime' do
+    education = FactoryGirl.build(:education)
+    it 'converts education start date to strftime' do
+      expect(education.pretty_date(education.start_date)).to eq '01/01/2001'
```

Comment:
I don't think I'm understanding you on this one... pretty_date expects an argument, which it seems like needs to be required because education has three dates (start, end, degree).  How would it know which date to prettify if we just do education.pretty_date?